packages/flutter/test/cupertino/context_menu_test.dart

```diff
@@ -241,27 +241,96 @@ void main() {
       expect(findStatic(), findsOneWidget);
     });
 
+    testWidgets('DecoyChild preserves the child color', (WidgetTester tester) async {
+      Widget getChild() {
+        return Container(
+          width: 150.0,
+          height: 150.0,
+          decoration: BoxDecoration(
+            color: CupertinoColors.activeBlue,
+            borderRadius: BorderRadius.circular(25.0),
+          ),
+        );
+      }
+      final Widget child = getChild();
+
+      await tester.pumpWidget(CupertinoApp(
+        home: CupertinoPageScaffold(
+          backgroundColor: CupertinoColors.black,
+          child: MediaQuery(
+            data: const MediaQueryData(size: Size(800, 600)),
+            child: Center(
+              child: CupertinoContextMenu(
+                actions: const <CupertinoContextMenuAction>[
+                  CupertinoContextMenuAction(
+                    child: Text('CupertinoContextMenuAction'),
+                  ),
+                ],
+                child: child
+              ),
+            )
+          )
+        ),
+      ));
+
+      /// test the child's shape and color
```

Comment:
These comments should have only 2 slashes and should start with a capital letter and end with a period.

Reply:
Used this style with all comments in the latest commit.